lv_sjpg.h

```diff
@@ -1,65 +1,51 @@
-/    It will contain multiple JPEG fragments all embedded into a single file with a custom header.
-/    This makes JPEG decoding easier using any JPEG library. Overall file size will be almost
-/    similar to the parent jpeg file. We can generate sjpeg from any jpeg using a python script
-/    provided along with this project.			
-/                                                                                     (by vinodstanur | 2020 )
-/    SJPEG FILE STRUCTURE
-/    --------------------------------------------------------------------------------------------------------------------------------
-/    Bytes                       |   Value                                                                                           |
-/    --------------------------------------------------------------------------------------------------------------------------------
-/                                                                                                                                    
-/    0 - 7                       |   "_SJPG__" followed by '\0'
-/
-/    8 - 13                      |   "V1.00" followed by '\0'       [VERSION OF SJPG FILE for future compatibiliby]
-/
-/    14 - 15                     |   X_RESOLUTION (width)            [little endian]
-/
-/    16 - 17                     |   Y_RESOLUTION (height)           [little endian]
-/
-/    18 - 19                     |   TOTAL_FRAMES inside sjpeg       [little endian]
-/
-/    20 - 21                     |   JPEG BLOCK WIDTH (16 normally)  [little endian]
-/
-/    22 - [(TOTAL_FRAMES*2 )]    |   SIZE OF EACH JPEG SPLIT FRAGMENTS   (FRAME_INFO_ARRAY)
-/
-/   SJPEG data                   |   Each JPEG frame can be extracted from SJPEG data by parsing the FRAME_INFO_ARRAY one time.
-/
-/----------------------------------------------------------------------------------------------------------------------------------
-/                   JPEG DECODER
-/                   ------------
-/	We are using TJpgDec - Tiny JPEG Decompressor library from ELM-CHAN for decoding each split-jpeg fragments.
-/	The tjpgd.c and tjpgd.h is not modified and those are used as it is. So if any update comes for the tiny-jpeg,
-/	just replace those files with updated files.
-/---------------------------------------------------------------------------------------------------------------------------------*/
-
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/*********************
+ *      INCLUDES
+ *********************/
+
 #include "stdint.h"
 #include <stdio.h>
+#include "lvgl.h"
+#include "tjpgd.h"
 
+/*********************
+ *      DEFINES
+ *********************/
+//#define LV_SJPG_FREE(x)                 free(x)
+//#define LV_SJPG_MALLOC(x)               malloc(x)
+#define TJPGD_WORKBUFF_SIZE             4096    //Recommended by TJPGD libray
+
+//NEVER EDIT THESE OFFSET VALUES
 #define SJPEG_VERSION_OFFSET            8
```

Comment:
Are these required in the header file? If not could you move it to the C file?

Reply:
Yes will move to the c.